components/codeflare/codeflare.go

```diff
@@ -71,6 +72,21 @@ func (c *CodeFlare) GetComponentName() string {
 	return ComponentName
 }
 
+func (c *CodeFlare) UpdateStatus(in *status.ComponentsStatus) error {
+	codeFlareStatus, err := deploy.GetReleaseVersion(in, deploy.DefaultManifestPath, ComponentName)
+
+	if err != nil {
+		in.CodeFlare = &status.CodeFlareStatus{}
+		return err
+	}
+
+	in.CodeFlare = &status.CodeFlareStatus{
+		ComponentStatus: codeFlareStatus,
```

Comment:
will unconditional setting this with return `err` and avoiding the first check make any difference?